packages/dev/docs/src/Layout.js

```diff
@@ -123,6 +123,11 @@ function Page({children, currentPage, publicUrl, styles, scripts, pathToPage}) {
   let title = currentPage.title + (!INDEX_RE.test(currentPage.name) || isBlog ? ` – ${pageSection}` : '');
   let hero = (parts.length > 1 ? HERO[parts[0]] : '') || heroImageHome;
   let heroUrl = `https://${TLD}/${currentPage.image || path.basename(hero)}`;
+  let githubLink = pathToPage;
```

Comment:
so the current url that the edit page bring us to is something like `.../tmp/{hash}/docs/@react-spectrum/docs/{the result of the url}` and the resulting string from this logic would be `docs/{the rest of the stuff}`. but i think that would still result in a 404 because the url to actually get to the docs is `packages/{the rest of the stuff}` so i think we'd want to replace `docs` with `packages`. 

you can see what i mean here. so this is the url that the edit button currently bring you to: https://github.com/adobe/react-spectrum/tree/main//tmp/38a4410cad4337bcf8bd79cde4df1582/docs/react-aria-components/docs/Button.mdx

whereas this is the url that we would want instead. 
https://github.com/adobe/react-spectrum/blob/main/packages/react-aria-components/docs/Button.mdx

and this is the url we would presumably get from this pr:
https://github.com/adobe/react-spectrum/tree/main/docs/react-aria-components/docs/Button.mdx

Reply:
Good catch, should be fixed now